### DISCUSSION TITLE
Citations showing the same name

### DISCUSSION THREAD

**Discussion by goallabuci** [greenelab/lab-website-template#132] [Q&A] [answered]
I added a citation in the source.yaml file with a title like this:
`- title: The title of my paper`

and when I reload the page, all papers show the same paper (namely, just the title of this new paper and empty metadata).

Any ideas what might be causing the issue?

**Comment by vincerubinetti** [comment, marked as answer]
Thank you for the report. This is a bug that I transferred to https://github.com/greenelab/lab-website-template/issues/133 and fixed in https://github.com/greenelab/lab-website-template/commit/a4b7d75404364d244bc1c13de73d1d7108ef89b9

To fix your repo most conveniently, look at that commit for what hotfix/patch to apply. Or, alternatively, [update your version of the template](https://github.com/greenelab/lab-website-template/wiki/Get-Started#update-the-template).
